src/parser/smt2/Smt2.g

```diff
@@ -900,15 +797,151 @@ sygusCommand [std::unique_ptr<CVC4::Command>* cmd]
       cmd->reset(new CheckSynthCommand(body));
     }
   | command[cmd]
- //   /* error handling */
- // | (~(CHECK_SYNTH_TOK))=> token=.
- //   { std::string id = AntlrInput::tokenText($token);
- //     std::stringstream ss;
- //     ss << "Not a recognized SyGuS command: `" << id << "'";
- //     PARSER_STATE->parseError(ss.str());
- //   }
   ;
 
+/** Reads a sygus grammar
+ *
+ * The resulting sygus datatype encoding the grammar is stored in ret.
+ * The argument sygus_vars indicates the sygus bound variable list, which is
+ * the argument list of the function-to-synthesize (or null if the grammar
+ * has bound variables).
+ * The argument fun is a unique identifier to avoid naming clashes for the
+ * datatypes constructed by this call.
+ */
+sygusGrammar[CVC4::Type & ret,
```

Comment:
you can remove the `CVC4::` prefixes everywhere

Reply:
Actually, the way the ANTLR C++ is generated, these CVC4:: are required.